app/src/main/java/de/tum/in/tumcampusapp/component/tumui/roomfinder/NavigationDetailsFragment.kt

```diff
@@ -89,13 +89,29 @@ class NavigationDetailsFragment : BaseFragment<Unit>(
         binding.locationName.text = navigationDetails.name
         binding.locationType.text = getCapitalizeType(navigationDetails.typeCommonName)
 
+        setupShareLocationButton(navigationDetails)
         setOpenInOtherAppBtnListener(navigationDetails)
 
         showNavigationDetailsProperties(navigationDetails)
 
         showAvailableMaps(navigationDetails)
     }
 
+    private fun setupShareLocationButton(navigationDetails: NavigationDetails) {
+        val url = "https://nav.tum.sexy/view/${navigationDetails.id}"
```

Comment:
nav.tum.app pls

Reply:
Since we serve at `nav.tum.de`,
I think this would be better:
```suggestion
        val url = "https://nav.tum.de/view/${navigationDetails.id}"
```